src/nft/components/profile/view/ProfilePage.tsx

```diff
@@ -53,8 +46,7 @@ const WALLET_COLLECTIONS_PAGINATION_LIMIT = 300
 const FILTER_SIDEBAR_WIDTH = 300
 const PADDING = 16
 
-export const ProfilePage = () => {
-  const { address } = useWalletBalance()
+export const ProfilePage = ({ address }: { address: string }) => {
```

Comment:
nit: you could use this hook instead of passing it to the component `const { account } = useWeb3React()`